package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "webpack-setup",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "private": true,
+  "scripts": {
+    "start": "webpack serve --mode development",
+    "build": "webpack --mode production"
```

Comment:
- [ ] I have seen that you added tests for comments and likes counters respectively, but Jest was not added under the script for running tests. Kindly share in the readme file how your project should be tested (the command users should run to test the project).